server/models/bucketlist.model.js

```diff
@@ -1,7 +1,26 @@
 const mongoose = require('mongoose');
 
 const bucketListSchema =  new mongoose.Schema({
-    name:{type: String, required: true}
+    name:{
```

Comment:
can we make this `title` instead? or what are your thoughts?

Reply:
title sounds good.